drivers/scheduler/scheduler.go

```diff
@@ -35,6 +35,11 @@ func (in *Context) DeepCopy() *Context {
 	return out
 }
 
+// GetNamespace returns the namespace for the context
+func (in *Context) GetNamespace() string {
```

Comment:
As I had mentioned offline, this actually breaks the scheduler abstraction, since there is no concept of "namespace" in DC/OS

Reply:
Do you prefer renaming this to GetID(). So this will be the context's unique ID. For k8s users, this ID is implicitly the namespace.